documentation/book/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='installing-your-own-ca-certificates-{context}']
+= Installing your own CA certificates
+
+This procedure will show you how to install your own CA certificates and private keys instead of using CA certificates and private key generated by the Cluster Operator.
+
+.Prerequisites
+
+* The Cluster Operator up and running
+* An `Kafka` resource within {ProductPlatformName}
+* Your own X.509 certificates and keys in PEM format for the cluster CA or clisters CA. These could be generated using `openssl` using a command such as:
++
+[source,shell,subs="+quotes"]
+openssl req -x509 -new -days _<validity>_ --nodes -out ca.crt -keyout ca.key
+
+
+.Procedure
+
+. Edit the `Kafka` resource for your cluster, configuring the `Kafka.spec.clusterCa` or `Kafka.spec.clientsCa` object (as appropriate) to _not_ use generated CAs:
++
+.Example fragment `Kafka` resource configuring the cluster CA to use certificates you supply for yourself
+[source,yaml]
+----
+kind: Kafka
+version: v1alpha1
+spec:
+  # ...
+  clusterCa:
+    generateCertificateAuthority: false
```

Comment:
This is a bit Off-Topic ... but did you implemented that when the secret is not generated by us we do not set the OwnerReference on it? So that we don't wipe it together with the cluster.

Reply:
Good point. No, but is that really what the user would expect?

Reply:
I think that when the users creates the secret we should not delete it. Anyway ... not critical. Might be just worth raising an issue / JIRA.